drizzlepac/haputils/catalog_utils.py

```diff
@@ -1479,10 +1477,11 @@ def identify_sources(self, **pars):
 
                 # Compute the ratio of big sources/islands using Custom/Gaussian kernel vs Rickerwavelet kernel
                 # This value can be used as a discriminant between overlapping point sources and nebulousity fields
-                ratio_cg2rw_bigsource = g_bs / rw_bs
+                if rw_bs > 0.0:
+                    ratio_cg2rw_bigsource = g_bs / rw_bs
```

Comment:
If 'rw_bs' == 0, the variable 'ratio_cg2rw_bigsource' never gets initialized with a value, yet it gets used later (line 1586).  On the face of it, this would cause an exception unless having 'rw_bs' == 0 somehow prevents the logic on line 1586 from ever being called.  Please clarify to insure no Exceptions get thrown.

Reply:
Oops.  Fixed in 9a8b0d5.